docs/releases/1.11.txt

```diff
@@ -205,7 +205,9 @@ Database backend API
 Miscellaneous
 -------------
 
-* ...
+* The :meth:`SyndicationFeed.latest_post_date <django.utils.feedgenerator.SyndicationFeed.latest_post_date>`
```

Comment:
Sorry I missed that, but `datetime.datetime.utcnow().replace(tzinfo=utc)` is independent of `USE_TZ`'s value (so I think). 

So it won't matter if `USE_TZ = False`.
I can add a test case though. Let me know if it is necessary.